index.html

```diff
@@ -234,6 +241,7 @@ <h1 class="aboutheader">
                     <button class="buttonaction">Get my resume</button>
                 </div>
             </div>
+            </main>
```

Comment:
- [ ] You should stick to the design as much as possible like color, Font, and background. Please use the same color for `h3` tag. Please see the image below for reference
- [ ] Also remove the border above the `Framework` to stick to the design
### You design
![your](https://user-images.githubusercontent.com/81727988/207804920-a4c2ea91-e8cc-43f0-8127-9640ae241c36.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/207804937-01b4bba9-4afa-48e3-a4e7-f4caafcb757e.png)